frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -348,6 +356,8 @@ public void run() {
     @UIEffect
     public void popupMenuWillBecomeInvisible(PopupMenuEvent popupMenuEvent) {
       graphTable.setRowSelectionAllowed(false);
+      // To selection (and thus the area where tooltip is shown) have again the possible smallest size
```

Comment:
```suggestion
      // So that the selection (and thus the area where tooltip is shown) has again the possible smallest size
```

Reply:
(`So that <subject> <verb 3rd person>` or `For <subject> to <verb infitive>...`)